internal/gengapic/gengapic.go

```diff
@@ -481,6 +489,34 @@ func (g *generator) emptyUnaryCall(servName string, m *descriptor.MethodDescript
 	return nil
 }
 
+func (g *generator) deadline(s, m string) {
+	t, ok := g.grpcConf.Timeout(s, m)
+	if !ok {
+		return
+	}
+
+	g.printf("if _, set := ctx.Deadline(); !set {")
```

Comment:
Nit: I think `ok` is usually used as the variable name? Probably doesn't matter.

Reply:
I'll go with `ok` to stick to common practice. Done.